docker_run.sh

```diff
@@ -1,46 +1,11 @@
 #!/usr/bin/env sh
 
-usage() {
-	printf "Usage: %s [--cpus <quantity-of-cpu>] [-h | --help]\n" "$(basename "$0")"
-}
-
-user_cpu_count=1
-cpu_count=0
-pwd=$(pwd)
-
-while [ $# -gt 0 ]; do
-	case "$1" in
-		"-h"|"--help") usage; exit 0 ;;
-		"--cpus") user_cpu_count=$2; shift 2 ;;
-		*) usage; exit 0 ;;
-	esac
-done
-
-case "$(uname)" in
-	"Linux") cpu_count=$(grep -c 'processor' /proc/cpuinfo) ;;
-	"Darwin") cpu_count=$(sysctl -n hw.logicalcpu) ;;
-	*) cpu_count=1 ;;
-esac
-
-if [ $user_cpu_count -gt $cpu_count ] || [ $cpu_count -eq 1 ]; then
-	user_cpu_count=$cpu_count
-fi
-
-if [ $user_cpu_count -eq 0 ]; then
-	user_cpu_count=$(echo "$cpu_count / 2" | bc)
-fi
-
 if [ ! -e database.db ]; then
 	touch ./database.db
 fi
 
 mkdir -p ./results/
 mkdir -p ./map/static/data/
 
```

Comment:
Add a check for the presence of the `markers.json` file to correctly launch the container with the map in case of a clean environment.
Like this: `[ -e ./map/static/data/markers.json ] || echo "{}" > ./map/static/data/markers.json`
Look here for details: https://asciinema.org/a/PIIUkbv2nOr9vf2e8x6zjzWxn

Reply:
Thanks! You are totally right. Also, this needs to be fxd on the backend application level.

Reply:
Fxd: b5c1cb8